File: C/PRAK110-2210817110005-MIZZUDDINABDISSALAM.c
#include <stdio.h>
#include <math.h>
int main(){
int alas = 5, tinggi = 12, A, B, C, K, L, P;
printf("Diketahui :\n");
printf("Alas = %d cm\n", alas);
printf("Tinggi = %d c,\n", tinggi);
printf("Jawab :\n");
A = tinggi;
printf("sisi A = %d cm\n", A);
C = alas;
printf("Sisi C = %d cm\n", C);
P = (A * A) + (C * C);
B = sqrt(P);
printf("Sisi B = %d cm\n", B);

K = A + B + C;
printf("Keliling = %d\n", K);
L = 0.5 * A * C;
printf("Luas = %d\n", L);
return 0;
}

File: Python/PRAK110-2210817110005-MIZZUDDINABDISSALAM.py
import math
print("Diketahui :")
a = 12
c = 5
print("Alas =", c,"cm")
print("Tinggi =", a,"cm\n")

print("Jawab :")
p = (a * a) + (c * c)
b = math.sqrt(p)
print("Sisi A =", a,"cm")
print("Sisi B =", round(b),"cm")
print("Sisi C =", c,"cm")
k = a + b + c
print("Keliling =", round(k),"cm")
l = 0.5 * a * c
print("Luas =", round(l),"cm")